src/Helper/SettingsHelper.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace Invoice\Helper;
+
+use Invoice\Services\SettingsService;
+
+/**
+ * Class SettingsHelper
+ *
+ * @author emmanouil.stafilarakis <emmanouil.stafilarakis@plentymarkets.com>
+ *
+ * @package Invoice\Helper
+ */
+class SettingsHelper
+{
+    /**
+     * The current settings
+     * 
+     * @var SettingsService
+     */
+    private $settings;
+    
+    /**
+     * The current plenty id.
+     * 
+     * @var int
+     */
+    private $plentyId;
+
+    /**
+     * SettingsHelper constructor.
+     * 
+     * @param SettingsService $settings The settings service
+     * @param int             $plentyId The plenty ID
+     */
+    public function __construct(SettingsService $settings, int $plentyId)
+    {
+        $this->settings = $settings;
+        $this->plentyId = $plentyId;
+    }
+
+    /**
+     * Get whether the country is active or not
+     * 
+     * @param int $countryId    The country ID to be checked.
+     * 
+     * @return bool
+     */
+    public function isCountryActive(int $countryId): bool
+    {
+        $countries = $this->settings->getShippingCountriesByPlentyId($this->plentyId);
+        return in_array($countryId, $countries);
+    }
+
+    /**
+     * Get whether or not the payment method has active countries.
+     * 
+     * @return bool
+     */
+    public function hasActiveCountries(): bool
+    {
+        $countries = $this->settings->getShippingCountriesByPlentyId($this->plentyId);
+        return count($countries) > 0;
+    }
+
+    /**
+     * Get the minimum amount for using the payment method.
+     * 
+     * @return float
+     */
+    public function minimumAmount(): float
+    {
+        return (float)$this->getSettingFloatValue('minimumAmount');
+    }
+    
+    /**
+     * Get the maximum amount for using the payment method.
+     * 
+     * @return float
+     */
+    public function maximumAmount(): float
+    {
+        return (float)$this->getSettingFloatValue('maximumAmount');
+    }
+    
+    /**
+     * Get the minimum amount for using the payment method.
+     * 
+     * @return float
+     */
+    public function minimumOrderCount(): int
+    {
+        return (int)$this->getSettingIntValue('quorumOrders');
+    }
```

Comment:
1. The annotation says return `float`
2. Is the int cast really necessary, because the method `getSettingIntValue` should always return an integer and there is also a cast in it, same question for the method with float.

Reply:
1. Copy&Paste Fehler aus `minimumAmount()`
2. Hast absolut recht.